iroh-relay/Cargo.toml

```diff
@@ -19,7 +19,7 @@ crate-type = ["lib", "cdylib"]
 workspace = true
 
 [dependencies]
-anyhow = { version = "1" }
+anyhow = { version = "1.0.98" }
```

Comment:
Needed for https://docs.rs/anyhow/latest/anyhow/struct.Error.html#method.into_boxed_dyn_error